### PR TITLE
Update documentation. Minor reformating on the fio-poarse-jsons.py

### PR DESCRIPTION
**Pull request by perezjosibm**
- Update documentation for the Test plan schema: workloads, benchmarks, cluster.
- Add some links from the main README.md

### CODE REVIEW

**Comment by perezjosibm** [comment]
Hi @sseshasa, would you be able to review thess minor updates to the documentation please? Many thanks in advance

File: docs/TestPlanSchema.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+The contents of the monitoring entity consists of :
+* a `nodes` (third level indentation) list of processes to monitor (by default the osd nodes), and 
+* an optional string `args` (third level indentation) to indicate the arguments to the monitoring tool.
+
+
+## `client_endpoints`
+
+The client_endpoints section consists of a non-empty list of collections, each associated to a 
+benchmark entity, and typically indicating  the driver for the benchmark. The client_endpoints, if 
+occur on a test plan, must be cross referenced by the benchmark section, and as such normally the
```
Comment (sseshasa) [inline docs/TestPlanSchema.md:70]: Consider changing "occur" to  "specified".

File: docs/Workloads.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# Workloads
```
Comment (sseshasa) [inline docs/Workloads.md:1]: Again the heading level doesn't seem to be rendered by github. You can check what's up with this.

File: docs/Workloads.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+A workload is the specification of a sequence of tests to be executed in the order given.
+Typically this involves a *range* of values for a specific benchmark argument. The most used is
+the *queue depth*. Depending of the benchmark, this can be expressed as a function of the number
+of jobs (or threads, or processes), such that the increase number of these causes a proportional
+increase in the I/O. Specifiying workloads in this way permits to generate *response latency curves*
+from the results.
+
+The workload feature are currently supported for `librbdfio` only.
```
Comment (sseshasa) [inline docs/Workloads.md:10]: s/are/is

File: tools/fio-parse-jsons/fio-parse-jsons.py [hunk 4]
```diff
@@ -39,48 +39,49 @@
...
+        "clat_stdev": "read/clat_ns",
+        "usr_cpu": "usr_cpu",
+        "sys_cpu": "sys_cpu",
+    },
+}
+
 
 def filter_json_node(next_branch, jnode_list_in):
-    """"
+    """ "
```
Comment (sseshasa) [inline tools/fio-parse-jsons/fio-parse-jsons.py:84]: I think the trailing double quote can be removed.

**Review by sseshasa — APPROVED** [review]
LGTM. Left minor comments.

File: docs/TestPlanSchema.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
+# Test plan schema
```
[the remark this answers is not in the document]
Comment (perezjosibm) [reply docs/TestPlanSchema.md:1]: Strange, it follows the correct syntax, not sure why the render is not showing it properly 🤔

[the remark this answers is not in the document]
Comment (sseshasa) [reply docs/TestPlanSchema.md:1]: This is an old thread but you can check if it applies...https://stackoverflow.com/questions/57136645/github-markdown-not-rendering-and-git-detecting-changes-when-no-changes

[the remark this answers is not in the document]
Comment (perezjosibm) [reply docs/TestPlanSchema.md:1]: I found some extra non-printable char that is not displayed on github between '#' and the text, removing it fixed the two cases

File: docs/Workloads.md (hunk 2 above)
Comment (perezjosibm) [reply docs/Workloads.md:1]: Fixed 👍

**Pull request merged**